resources/assets/components/CampaignIdSingle/index.js

```diff
@@ -82,13 +82,11 @@ class CampaignIdSingle extends React.Component {
             <p>–</p>
           )}
           <h4>Start Date</h4>
-          <p>
-            {format(new Date(campaign.start_date.replace(/-/g, '/')), 'M/d/yy')}
-          </p>
+          <p>{format(parse(campaign.updated_at), 'M/d/YY')}</p>
           <h4>End Date</h4>
           <p>
             {campaign.end_date
-              ? format(new Date(campaign.end_date.replace(/-/g, '/')), 'M/d/yy')
+              ? format(parse(campaign.updated_at), 'M/d/YY')
```

Comment:
Just noticed this when clicking through from #858 – these were accidentally changed from `campaign.end_date` to `campaign.updated_at` (and same thing on the "Start Date" line above & "Created At" line below).

Reply:
@DFurnes yup - fixed that [here](https://github.com/DoSomething/rogue/pull/858/files#diff-0e7fbc8575f3965559b364378e6d6e54R144)!